sdks/java/io/file-schema-transform/src/main/java/org/apache/beam/sdk/io/fileschematransform/FileWriteSchemaTransformProvider.java

```diff
@@ -129,7 +136,11 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
                                   .build()))
               .setRowSchema(OUTPUT_SCHEMA);
 
-      return PCollectionRowTuple.of(OUTPUT_TAG, output);
+      if (files.has(ERROR_TAG)) {
+        return PCollectionRowTuple.of(OUTPUT_TAG, output).and("error", files.get(ERROR_TAG));
```

Comment:
can we have a constant for "error" similar to OUTPUT_TAG. Let's change it all the places. Using hardcoded strings is not a good practice and makes refactoring difficult.

Reply:
Done